test/flex.js

```diff
@@ -62,7 +59,7 @@ test('flex invalid', async (t) => {
 });
 
 test('flex shrink/grow', async (t) => {
-  const boundsRangeArray  = getNumericArrayInRange(flexGrowShrink[0], flexGrowShrink[1]);
+  const boundsRangeArray  = getNumericArrayInRange(0, 5);
```

Comment:
Could you rename the constant - it was a bit confusing with the PR title :)

Reply:
Addressed here https://github.com/warp-ds/drive/pull/62/files#diff-4523a55c502753c1e3fbb6cd0db39d893b95f593d8166e4822dd76ffc005247bR63